features/docs/defining_steps/ambiguous_steps.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Ambiguous Steps
+ 
+  Scenario: Ambiguous steps
+
+    Given a file named "features/ambiguous.feature" with:
+    """
+    Feature:
+
+      Scenario: test 1
+      * a step
+      * an ambiguous step
+
+      Scenario: test 2
```

Comment:
I'm wondering why we need this second scenario. I'd find it easier to read this example if it wasn't there. Do we need it to illustrate the behaviour?